designer/server/src/main/scala/pl/touk/nussknacker/ui/process/test/ScenarioTestService.scala

```diff
@@ -70,23 +70,23 @@ class ScenarioTestService(testInfoProviders: ProcessingTypeDataProvider[TestInfo
     } yield rawTestData
   }
 
-  def performTest[T](idWithCategory: ProcessIdWithNameAndCategory,
+  def performTest[T](processIdWithName: ProcessIdWithName,
```

Comment:
This place is a bit more tricky than I expected. In this form, it's possible to replace category on FE, and then use it to test components one does not have access to...